docs/basics/connection-approval.md

```diff
@@ -153,6 +205,7 @@ The server assigns the `ConnectionApprovalCallback` when it spawns the in-scene
 
 ### Step 2: Copy the Alternate Player Prefab's GlobalObjectIdHash Value
 ![Copy-GlobalObjectIdHash](images/CopyGlobalObjectIdHash.png)
+
 In order to populate the `AlternatePlayerPrefabs` list:
 - Open the scene containing the in-scene placed `NetworkObject` that the `ConnectionApprovalCallback` is attached to.
 - Find each alternate player prefab you wish to add to the list, select the prefab (but don't open it), and copy the GlobalObjectIdHash value by right-clicking and selecting "copy".
```

Comment:
To populate the `AlternatePlayerPrefabs` list:
1. Open the scene containing the in-scene placed `NetworkObject` to which the `ConnectionApprovalCallback` is attached.
2. Find each alternate player prefab you wish to add to the list, select the prefab (but don't open it), and copy the `GlobalObjectIdHash` value by right-clicking and selecting **Copy**.